src/platform_impl/linux/wayland/window/state.rs

```diff
@@ -157,8 +161,14 @@ impl WindowState {
             .fractional_scaling_manager
             .as_ref()
             .map(|fsm| fsm.fractional_scaling(window.wl_surface(), queue_handle));
+        let blur = winit_state
+            .kwin_blur_manager
+            .as_ref()
+            .map(|kbm| kbm.blur(window.wl_surface(), queue_handle));
```

Comment:
What's the reason it's done like that? Can't we create an object when we actually will set a `blur`?